### PR TITLE
Status Bar attached to readme.md file

### PR DESCRIPTION
**Pull request by SAHU-01**
the repository status of issues,forks,stars,pr ,etc has been attached in readme.md file

### CODE REVIEW

**Review by chebroluharika — APPROVED** [review]
LGTM

**Review by chebroluharika — CHANGES_REQUESTED** [review]
Remove style-social for line5 and line 6

**Comment by SAHU-01** [comment]
ok
I'll do that right away

**Comment by chebroluharika** [comment]
I think it would be good if you can add badges for no of contributors, no of downloads from Pypi, Pypi version, GitHub license

**Comment by SAHU-01** [comment]
Sure I'll add them now!
But the repo doesn't show a license, so you can add a license and let me know I'll add that.

**Comment by SAHU-01** [comment]
yup got that i'll add it now thanks

**Comment by chebroluharika** [comment]
Hi @SAHU-01, give package name as flow2ml(all in small letters) and btw there is no LICENSE attached in repo, though we have license.txt, it just has copyright. Its my bad, that I didn't notice it.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 # Flow2ML
+
+[![Issues](https://img.shields.io/github/issues/flow2ml/Flow2ML)](https://github.com/flow2ml/Flow2ML/issues)
+[![PRs](https://img.shields.io/github/issues-pr-raw/flow2ml/Flow2ML)](https://github.com/flow2ml/Flow2ML/pulls)
+[![Forks](https://img.shields.io/github/forks/flow2ml/Flow2ML?)](https://github.com/flow2ml/Flow2ML/network/members) 
+[![Stars](https://img.shields.io/github/stars/flow2ml/Flow2ML?)](https://github.com/flow2ml/Flow2ML/stargazers)
+[![Contributors](https://img.shields.io/github/contributors/flow2ml/Flow2ML)](https://github.com/flow2ml/Flow2ML/graphs/contributors)
+[![PyPi Version](https://img.shields.io/pypi/v/pypi-download-stats.svg)]
```
Comment (chebroluharika) [inline README.md:8]: Remove extra square bracket at last

Comment (SAHU-01) [reply README.md:8]: ok

Comment (SAHU-01) [reply README.md]: it's done now sorry I did not notice [] were comeing extra

**Review by chebroluharika — APPROVED** [review]
LGTM

**Pull request merged**